parsec/backend/postgresql/user_queries/find.py

```diff
@@ -88,6 +108,32 @@ async def query_find(
     return results, len(all_results)
 
 
+@query()
+async def query_retrieve_human(
+    conn, organization_id: OrganizationID, arguments: List[Tuple[str, str, str]]
+) -> HumanFindResultItem:
```

Comment:
It seems that in can return None, so the return type shall be `Optional[HumanFindResultItem]`